blocked-edges/4.14.0-ARODNSWrongBootSequence.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.0
+from: .*
+url: https://issues.redhat.com/browse/OCPBUGS-35300
```

Comment:
nit: for OCP-core bugs, we often link an impact statement ([template][1]) that steps back a layer and tries to explain the issue to a cluster-admin considering the impacted update.  That cluster admin will likely want a clear picture of how to decide "is my cluster actually exposed?", but they probably don't care about things like *Steps to Reproduce* (QE trying to verify the bug will care very much about those steps).

But for an ARO-specific risk like this that clearly says "ARO" in the `name`, `message`, `matchingRules`, and linked URI, :shrug:, y'all are free to have whatever wording in that linked doc, and you can always pivot later if you change your minds about the focus of the doc you link.

[1]: https://github.com/openshift/enhancements/blob/b9df937d9fb86840d54e8faac77a2c74bbb804c1/enhancements/update/update-blocker-lifecycle/README.md#impact-statement-request

Reply:
I'll add it to my script for future use. Will Skip this time.